sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md

```diff
@@ -203,22 +203,11 @@ dataFeed.IngestionSettings = new DataFeedIngestionSettings()
     IngestionStartTime = DateTimeOffset.Parse("2020-01-01T00:00:00Z")
 };
 
-Response<string> response = await adminClient.CreateDataFeedAsync(dataFeed);
+Response<DataFeed> response = await adminClient.CreateDataFeedAsync(dataFeed);
 
-string dataFeedId = response.Value;
-
-Console.WriteLine($"Data feed ID: {dataFeedId}");
-```
-
-Note that only the ID of the data feed is known at this point. You can perform another service call to `GetDataFeedAsync` or `GetDataFeed` to get more information, such as status, created time, the list of administrators, or the metric IDs.
-
-```C# Snippet:GetDataFeedAsync
-string dataFeedId = "<dataFeedId>";
-
-Response<DataFeed> response = await adminClient.GetDataFeedAsync(dataFeedId);
-
-DataFeed dataFeed = response.Value;
+DataFeed createdDataFeed = response.Value;
 
+Console.WriteLine($"Data feed ID: {createdDataFeed.Id}");
```

Comment:
I'm not sure if I'm following. The sample is already updated. Previously I was linking two samples (Create and Get). I have removed the reference to the `GetDataFeed` sample and updated the `CreateDataFeed` sample accordingly. We're printing extra info, including the Metric IDs.

Reply:
So my confusion is that `createdDataFeed ` is only used to print the `id`. If the idea is to show that it has more data then consider using that type instead of `dataFeed` (when it makes sense)
ie.
```
Console.WriteLine($"Data feed ID: {createdDataFeed.Id}");
Console.WriteLine($"Data feed status: {createdDataFeed.Status.Value}");
Console.WriteLine($"Data feed created time: {createdDataFeed.CreatedTime.Value}");

```

Reply:
I see. Here's a snippet of the README:
```cs
DataFeed createdDataFeed = response.Value;

Console.WriteLine($"Data feed ID: {createdDataFeed.Id}");
Console.WriteLine($"Data feed status: {dataFeed.Status.Value}");
Console.WriteLine($"Data feed created time: {dataFeed.CreatedTime.Value}");

Console.WriteLine($"Data feed administrators:");
foreach (string admin in dataFeed.Administrators)
{
    Console.WriteLine($" - {admin}");
}

Console.WriteLine($"Metric IDs:");
foreach (DataFeedMetric metric in dataFeed.Schema.MetricColumns)
{
    Console.WriteLine($" - {metric.MetricName}: {metric.MetricId}");
}

Console.WriteLine($"Dimension columns:");
foreach (DataFeedDimension dimension in dataFeed.Schema.DimensionColumns)
{
    Console.WriteLine($" - {dimension.DimensionName}");
}
```

I'm using `dataFeed` instead of `createdDataFeed` by accident. Good catch.

Reply:
Fixed.